package-examples/helloworld/README.md

```diff
@@ -0,0 +1,53 @@
+# helloworld
+
+Sample helloworld package.  Showcases what a bare bones package is which doesn't do anything beyond declaring the current directory as a `kpt` package. 
+
+## Steps
+
+This is a simple workflow on a package that requires no configuration or
+customization.
+
+1. [Fetch the package](#fetch-the-package)
+2. [View the package contents](#view-the-package-contents)
+3. [Apply the package](#apply-the-package)
+
+### Fetch the package
+
+Get the example package on to local using `kpt pkg get`
+
+  $ kpt pkg get https://github.com/GoogleContainerTools/kpt.git/package-examples/helloworld
+
+    fetching package /package-examples/helloworld from https://github.com/GoogleContainerTools/kpt to helloworld
+
+### View the package contents
+
+List the package contents in a tree structure.
+
+  $ kpt pkg tree helloworld/
+
+    helloworld
+    ├── [deploy.yaml]  Deployment helloworld-gke
+    └── [service.yaml]  Service helloworld-gke
+
+### Apply the package
+
+Initialize the inventory object:
+
+  $ kpt live init helloworld
+
+    namespace: default is used for inventory object
+    Initialized: helloworld/inventory-template.yaml
+
+
+Apply all the contents of the package recursively to the cluster
+
+  $ kpt live apply helloworld 
+
+    service/helloworld-gke created
+    deployment.apps/helloworld-gke created
+    2 resource(s) applied. 2 created, 0 unchanged, 0 configured, 0 failed
+    0 resource(s) pruned, 0 skipped, 0 failed
+
+[tree]: ../../../site/reference/pkg/tree
+[apply]: ../../../site/reference/live/init
+[apply]: ../../../site/reference/live/apply
```

Comment:
Same marker is pointing to different links. Will this work ?

Reply:
I don't see these are being referenced in the file. Were these added by mistake ? If so, please remove.

Reply:
updated, but this is modeled after your wordpress readme.
 `live init` and `live apply` are referenced in *Apply the package*

Reply:
I thought you wanted to add handy links for commands used in the file.

Reply:
These are not visible unless you reference them in the text using []. Removed them. Thanks for pointing out as there are such links in my PR too. Will clean them up.